src/app/app.component.html

```diff
@@ -1,3 +1,4 @@
 <app-header></app-header>
+<app-breadcrumbs *appIfAuthenticated="this.authService.isLoggedIn"></app-breadcrumbs>
```

Comment:
You shouldn't expose the service in the template. You can create a getter in the component and use it in the template.